### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by Sparta-9**

### CODE REVIEW

**Review by thehairy — COMMENTED** [review]
Useless as nearly every PR in the last 12 hours.

**Comment by JacobMGEvans** [comment]
> Useless as nearly every PR in the last 12 hours.

Anyone trying to help and improve is always welcomed.

File: README.md [hunk 1]
```diff
@@ -104,7 +104,7 @@ a complete authority.
 ### Exercises
 
 - `src/exercise/00.md`: Background, Exercise Instructions, Extra Credit
-- `src/exercise/00.js`: Exercise with Emoji helpers
+- `src/exercise/00.js`: Exercise with Emoji helper's
```
Comment (JacobMGEvans) [inline README.md:107]: Grammatically I don't think this helper has possession of anything.

**Comment by thehairy** [comment]
> > Useless as nearly every PR in the last 12 hours.
> 
> Anyone trying to help and improve is always welcomed.

Nobody in the last few hours tried to help you. They just want to get as much PR's as possible for Hacktoberfest.
So please don't fall in that trap and just close those spammy PRs.

File: README.md [hunk 2]
```diff
@@ -123,7 +123,7 @@ emoji characters are here to help you.
 - **Matthew the Muscle** 💪 will indicate what you're working with an exercise
 - **Chuck the Checkered Flag** 🏁 will indicate that you're working with a final
   version
-- **Marty the Money Bag** 💰 will give you specific tips (and sometimes code)
+- **Marty the Money Bag** 💰 will give you a specific tips (and sometimes code)
```
Comment (JacobMGEvans) [inline README.md:126]: The grammar without the "a" was correct, "a" would for a singular tip, not plural "tips" 😄

**Comment by JacobMGEvans** [comment]
can be safely closed @kentcdodds

**Comment by kentcdodds** [comment]
🤖 This is Hacktoberfest spam and will be automatically deleted.

If you feel this was in error, please comment.

**Pull request closed without merging**